.github/copy-pr-bot.yaml

```diff
@@ -3,5 +3,5 @@
 
 enabled: true
 additional_trustees:
-- ahendriksen
-- gonzalobg
+  - ahendriksen
+  - gonzalobg
```

Comment:
Do we want to add @robertmaynard too?

Reply:
I believe I am in a team that can trigger PRs